recipes/mss/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.2.0" %}
+
+package:
+    name: mss
+    version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        # mss import pyproj from basemap and that does not exist in 1.0.8
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
+        - paste
+        - pillow
+        - python-dateutil
+        - pytz
+        - requests
+        - scipy
+        - six
+        - tk 8.5.*
+    run:
+        - python
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
+        - paste
+        - pillow
+        - python-dateutil
+        - pytz
+        - requests
+        - scipy
+        - six
+        - tk 8.5.*
+        # FIXME: Workaround until https://github.com/conda-forge/matplotlib-feedstock/issues/36 is fixed.
```

Comment:
> For now I want to add the modified yaml to our mss repository.

Sure. So a comment here would help your users to distinguish them, right? I will do it like:

`Relaxed versions in the meta.yaml for conda-forge)`

Is that OK?

> (I modify it that it could work with defaults too for now

That is not an easy task! The maintainability will be hard. But I understand the reasons.

> A lot of the tools here makes it much easier to have faster builds. All that testing makes it much easier for us.

Glad to hear tit :smile: